yaptide/admin/db_manage.py

```diff
@@ -55,7 +57,10 @@ def list_users(**kwargs):
     if con is None or metadata is None or engine is None:
         return None
     users = db.Table(TableTypes.USER.value, metadata, autoload=True, autoload_with=engine)
-    query = db.select([users])
+    yaptide_users = db.Table(TableTypes.YAPTIDEUSER.value, metadata, autoload=True, autoload_with=engine)
+    joined_users = users.join(yaptide_users, users.c.id == yaptide_users.c.id)
```

Comment:
What do you mean by joined users ? Add a comment explaining this code. Do you mean local users and keycloak users ? 
Why do you make join based on having the same id ?

Reply:
Outdated